src/components_redux/Input.js

```diff
@@ -16,10 +15,9 @@ class Input extends Component {
     const style = { width: 550 };
     this.setState({ style });
   };
-  handleChangeShow = selectedValue => {
+  handleChangeShow = value => {
```

Comment:
Nothing to change, more as an info, you didn't needed to change the name of the variable here. This is an own function and is independent from names in Radios.